markdown/introduction.md

```diff
@@ -5,6 +5,18 @@ Nav: home.nav
 
 TableOfContents
 
+## {id="what"} What does it, really?
+
+What does it mean that "Bedrocklinux allows to run a linux system that is composed of different linux distributions"? It means, the system boots with a bootloader, a kernel, and an init system that can be independently selected from the "bedrocked" distributions, and can then freely run userland packages that are installable side-by-side from the different distributions.
```

Comment:
I think this organization is potentially confusing/misleading in several ways:

- Outside of `pmm`, Bedrock doesn't really understand packages.  In principle, it should work with components of distros that don't organize things with packages.
- While I can see room for debate with the bootloader and kernel, the init is definitely a userland package.  Listing the two as distinct is odd.
- Giving focus to the bootloader, kernel, and init, but then grouping everything else under "userland package" just raises ambiguity about what things other than bootloader, kernel, and init Bedrock can make work across distro boundaries.
- I don't know what you mean by "bedrocked."

Reply:
Thanks! Good points of course. I'll see what I can do with my current understanding.

With "bedrocked" I wanted to paraphrase the hijack/fetch into their strata/roots. I had used "installed" there, but wasn't too happy with it.